src/frontend/Komponenter/Journalforing/JournalforingAdmin.tsx

```diff
@@ -116,60 +101,111 @@ export const JournalforingAdmin: React.FC = () => {
+                        hentFagsak={hentFagsak}
+                    />
+                )}
+            </DataViewer>
+            <DataViewer response={{ journalResponse, fagsak }}>
+                {({ journalResponse, fagsak }) => (
+                    <SideLayout className={'container'}>
+                        <Sidetittel>Opprett ny behandling for journalpost</Sidetittel>
+                        <Blokk>
+                            <Normaltekst>
+                                Her kan du opprette en ny behandling med søknadsdata for en
+                                journalpost som er ferdigstilt
+                            </Normaltekst>
+                        </Blokk>
+                        <Blokk>
+                            <Brukerinfo personIdent={journalResponse.personIdent} />
+                        </Blokk>
+                        <Blokk>
+                            <Systemtittel>Journalpost</Systemtittel>
+                            <Normaltekst>{stønadstypeTilTekst[fagsak.stønadstype]}</Normaltekst>
+                            <Normaltekst>
+                                JournalpostID: {journalResponse.journalpost.journalpostId}
+                            </Normaltekst>
+                            <Normaltekst>
+                                Status:{' '}
+                                {journalstatusTilTekst[journalResponse.journalpost.journalstatus]}
+                            </Normaltekst>
+                            <Normaltekst>
+                                Dato mottatt:{' '}
+                                {formaterIsoDatoTid(journalResponse.journalpost.datoMottatt)}
+                            </Normaltekst>
+                        </Blokk>
+                        <Blokk>
+                            <Systemtittel>Behandlingstype</Systemtittel>
+                            {nyBehandlingstype && (
+                                <Normaltekst>
+                                    {behandlingstypeTilTekst[nyBehandlingstype]}
+                                </Normaltekst>
+                            )}
+                        </Blokk>
+
+                        <Hovedknapp
+                            onClick={() => sendInn(journalResponse, fagsak.id)}
+                            spinner={senderInn}
+                        >
+                            Opprett behandling
+                        </Hovedknapp>
+                        {!journalResponse.harStrukturertSøknad && (
+                            <AlertStripeAdvarsel>
+                                Kan ikke finne en digital søknad på denne journalposten.
+                            </AlertStripeAdvarsel>
                         )}
-                    </Blokk>
+                        {feilmelding && <AlertStripeFeil>{feilmelding}</AlertStripeFeil>}
+                    </SideLayout>
+                )}
+            </DataViewer>
+        </>
+    );
+};
+
+const UtledEllerVelgFagsak: React.FC<{
```

Comment:
Denne skal også brukes for "vanlig" journalføring, så kommer til å trekke den ut i egen fil.